quipucords/fingerprinter/runner.py

```diff
@@ -508,9 +513,12 @@ def _process_source(self, source):
             server_id = source.get("server_id")
             source_type = source.get("source_type")
             source_name = source.get("source_name")
-            if fact.get("cluster") and source_type == DataSources.OPENSHIFT:
+            if (
+                fact.get("cluster") and source_type == DataSources.OPENSHIFT
+            ) or source_type == DataSources.ACS:
```

Comment:
this will cause catastrophic failures to discovery unless https://issues.redhat.com/browse/DISCOVERY-430 is implemented.

the reason this hack works for openshift is because we DO generate a deployment reports for it, we just don't add cluster data